devops/github_issue_alert/autokitteh.yaml

```diff
@@ -0,0 +1,27 @@
+# This YAML file is a declarative manifest that describes the setup of
+# an AutoKitteh project that monitors comments on GitHub issues.
+
+version: v1
+
+project:
+  name: github_issue_alert
+
+  connections:
+    - name: slack_conn
+      integration: slack
+    - name: GitHub_conn
+      integration: github
+
+  triggers:
+    - name: on_issue_comment
+      event_type: issue_comment
+      connection: GitHub_conn
+      call: program.py:on_issue_comment
+    - name: on_issue_created
+      event_type: issues
+      connection: GitHub_conn
+      call: program.py:on_issue_created
+
+  vars:
+    - name: SLACK_CHANNEL
+      value: GitHub_Issues
```

Comment:
1. I understand why you're putting this section last - like in the UI - but you should put it above connections, like in all other projects because it usually needs manual inspection/modification, and the other sections usually don't
2. See #157 
3. `GitHub_Issues` isn't a valid channel name? Use `github-issues` (`_` is valid but less common as a replacement for whitespaces in Slack)